### PR TITLE
Add Enkrypt and MEW Wallet DApp browser as SDKs

### PR DESCRIPTION
**Pull request by gamalielhere**
### Description
Adds Enkrypt and MEW Wallet DApp browser as SDKs
Rename current mew package to mew connect

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

**Comment by aaronbarnardsound** [comment]
Hey @gamalielhere thanks for this, we'll try to get a look at this one soon 😄

**Comment by Adamj1232** [comment]
Hi @gamalielhere! Do you mind if I clone down your fork and make some updates? Might be easier than going back and forth through comments 😃

**Comment by gamalielhere** [comment]
@Adamj1232 go for it! let me know if you need anything from our end

**Comment by Adamj1232** [comment]
@gamalielhere Excellent will do, thank you!
Have the current changes been verified through testing on your side?

**Comment by gamalielhere** [comment]
@Adamj1232 yes. I tested it using the demo folder and importing the packages directly.

That access method will be deprecated at some point in the near future. Also, the new mew-wallet package uses a web3 injection access method via the MEW wallet app's DApp Browser.

**Comment by Adamj1232** [comment]
@gamalielhere Ah gotcha, so we can deprecate the MEW package with the release of the MEW-wallet package?
Also, I have local changes made but can't push to your forked branch.

**Comment by Adamj1232** [comment]
@gamalielhere friendly ping - any updates on allowing push access to your repo or the question above?

**Comment by gamalielhere** [comment]
@Adamj1232 I might have to close and open a new PR to enable write access to my changes it seems. Regarding the package, yes it can be deprecated in favor of this PR.

**Comment by Adamj1232** [comment]
@gamalielhere thank you for the push access! I have pushed some changes but will need your help to clear the build errors produced when running `yarn build`. 
Let me know when those are cleared and I can start testing, cheers!

**Comment by gamalielhere** [comment]
@Adamj1232 sync'd and fixed the `yarn build` errors.

**Comment by Adamj1232** [comment]
@gamalielhere thank you! Have you tested this using the internal project demo?

File: packages/injected/src/wallets.ts [hunk 1]
```diff
@@ -578,6 +578,47 @@ const bitski: InjectedWalletModule = {
...
+
+    const addListener: SimpleEventEmitter['on'] = window.enkrypt.providers.ethereum.on.bind(
+      window.enkrypt.providers.ethereum
+    )
+
+    window.enkrypt.providers.ethereum.on = (event, func) => {
+      // intercept chainChanged event and format string
+      if (event === 'chainChanged') {
+        addListener(event, (chainId: ChainId) => {
+          console.log(chainId)
```
Comment (Adamj1232) [inline packages/injected/src/wallets.ts:597]: sneaky log

File: packages/injected/src/icons/mew-wallet.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export default `
```
Comment (Adamj1232) [inline packages/injected/src/icons/mew-wallet.ts:1]: This svg isnt needed and can be removed as MEW is not an injected-wallet package option

File: packages/enkrypt/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# @web3-onboard/enkrypt
+
+## Wallet module for connecting Mew wallet to web3-onboard
```
Comment (Adamj1232) [inline packages/enkrypt/README.md:3]:
```suggestion
## Wallet module for connecting Enkrypt wallet to web3-onboard
```

File: packages/mew-wallet/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# @web3-onboard/mew-wallet
+
+## Wallet module for connecting Mew wallet to web3-onboard
```
Comment (Adamj1232) [inline packages/mew-wallet/README.md:3]:
```suggestion
## Wallet module for connecting Mew wallet through web3-onboard
```

File: packages/enkrypt/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# @web3-onboard/enkrypt
+
+## Wallet module for connecting Enkrypt wallet to web3-onboard
```
Comment (Adamj1232) [inline packages/enkrypt/README.md:3]:
```suggestion
## Wallet module for connecting Enkrypt wallet through web3-onboard
```

**Review by Adamj1232 — APPROVED** [review]
Good work! Thank you for collaborating on this!

**Review by Adamj1232 — APPROVED** [review]
Excellent work and fun collaborating, thanks again!

**Pull request merged**